_posts/blog/2019-08-22-2019-08-22-lab-meeting.md

```diff
@@ -0,0 +1,120 @@
+
+There are lots of updates here, and some really fascinating questions that we didn't manage to answer, but please do scroll to the end to see a few of Ruslan Yermakov's excellent brain visualisations for the `scona`🍪 (Structural Covariance Network Analyses) project ([https://github.com/WhitakerLab/scona](https://github.com/WhitakerLab/scona)).
+
+## Celebrations and cool things to share
+
+**Patricia** is starting a new job in September and her new team is running a survey on FAIR data policies and practices.
+It would be great for everyone to complete it and share widely!
+* [https://www.fairsfair.eu/fairsfair-open-consultation-fair-data-policies-and-practices](https://www.fairsfair.eu/fairsfair-open-consultation-fair-data-policies-and-practices)
+
+**Maxine** is back on track with thesis writing after a rough few weeks of re-running _all_ of her analyses because there was a mistake at the very start 😱
+This week she's writing new analyses (rather than re-writing the chapters that had to be updated).
+Kirstie is wildly proud that a whole PhD can be re-run in just a few weeks!
+Reproducible research FTW ✨ 🙌 🚀 😻
+
+In contrast, **Josh** has been busy incorporating feedback for his thesis _proposal_.
+He also took a break from his PhD by writing a blog on [Four things I wish I'd known before starting a PhD](https://medium.com/josh-cowls/four-things-i-wish-id-known-before-starting-a-phd-9e80f89ae04e).
+(With the excellent subtitle: _Some Of Which I Probably Could Have Guessed And Most Of Which I Was Probably Told At Some Point, But Now Know For Sure._)
+
+**Sarah** is much more organised with her post-it to-do list method 💖
+
+**Christina** is working on a fascinating project thinking about the ethics of machine learning in children's social care.
+Good luck with the tight deadline Christina!!
+
+**Louise** shared a new colourmap from Google: [Turbo: An improved rainbow colormap for visualisation](https://ai.googleblog.com/2019/08/turbo-improved-rainbow-colormap-for.html).
+The linked blog is great discussion of the pros and cons of rainbow colourmaps 🌈
+
+**Yini** is "Learning Python the Hard Way" (via [the book](https://smile.amazon.co.uk/Learn-Python-Hard-Way-Introduction/dp/0321884914/ref=smi_www_rco2_go_smi_g8682124849?_encoding=UTF8&%2AVersion%2A=1&%2Aentries%2A=0&ie=UTF8) by Zed A Shaw recommended by Sarah and Georgia.
+She found this video on youtube really helpful: [https://www.youtube.com/watch?v=9k8s8r5DXNU](https://www.youtube.com/watch?v=9k8s8r5DXNU).
+
+Yini also recommended the "Tell me a story" point of view article by Joshua Sanes at eLife (doi: [10.7554/eLife.50527](https://doi.org/10.7554/eLife.50527)).
+Sanes suggests starting with the introduction and results sections of a paper and leaving the figures to later.
+
+(Side note: This is not Kirstie's style at all, she's super into figures to tell the story, but its worth reading and seeing what fits your workflow best!)
+
```

Comment:
It is really coooool and logical to discribe a research only using 1 or 2 figures!